wasm/leb128/read.go

```diff
@@ -0,0 +1,73 @@
+package leb128
+
+import (
+	"io"
+)
+
+func ReadVarUintSize(r io.Reader) (res uint32, size uint, err error) {
+	byte := make([]byte, 1)
+	var shift uint = 0
+	for {
+		_, err := r.Read(byte)
+		if err != nil {
+			return 0, 0, err
+		}
+		size++
+
+		cur := uint32(byte[0])
+		res |= (cur & 0x7f) << (shift)
+		if cur&0x80 == 0 {
+			return res, size, nil
+		}
+		shift += 7
+	}
+}
+
+func ReadVarUint(r io.Reader) (uint32, error) {
```

Comment:
please document `ReadVarUint`.
it may just refer to `ReadVarUintSize`, and/or explaining what is the difference with the latter.

same comment than above about `32/64`.